examples/nuxt/README.md

```diff
@@ -0,0 +1,27 @@
+# @adyen/adyen-web v6 + Nuxt3
+
+### Steps to run the project:
+
+1. Install the project dependencies: `npm install`
+2. Edit the `nuxt.config.ts` file and add there your account details:
+
+Example:
+
+```
+runtimeConfig: {
```

Comment:
I think is better to not use nuxt.config.ts to write secrets. This is easy to forget and commit exposing the API key and other secrets. 
I used the .env file like this for testing: https://nuxt.com/docs/guide/going-further/runtime-config#example